src/manifold/module/hull.cpp

```diff
@@ -0,0 +1,65 @@
+// Copyright 2024 The Manifold Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "QuickHull.hpp"
+#include "manifold.h"
+#include "par.h"
+
+using namespace manifold;
+
+/**
+ * Compute the convex hull of a set of points. If the given points are fewer
+ * than 4, or they are all coplanar, an empty Manifold will be returned.
+ *
+ * @param pts A vector of 3-dimensional points over which to compute a convex
+ * hull.
+ */
+Manifold Manifold::Hull(const std::vector<glm::vec3>& pts) {
```

Comment:
@fire here is the implementation of these hull functions, which lives in the `hull` target. Without that linked in, these methods (defined in `manifold.h`) will fail to link if called.

Reply:
I expected this api to be in a .h header file. Strange.

Reply:
The API is defined in `manifold.h`, this is the implementation.